modules/renter/persist.go

```diff
@@ -235,17 +242,14 @@ func (r *Renter) load() error {
 	}
 
 	// Load contracts, repair set, and entropy.
-	data := struct {
-		Tracking  map[string]trackedFile
-		Repairing map[string]string // COMPATv0.4.8
```

Comment:
We shouldn't need this anymore, anyone upgrading from v0.4.8 is not going to have working files at this point.